package.json

```diff
@@ -35,10 +35,13 @@
     "graphql-tools": "^3.1.1",
     "graphql-yoga": "^1.16.2",
     "http-proxy": "^1.17.0",
-    "prisma-binding": "^2.1.5",
-    "react": "^16.5.1",
+    "jquery": "^3.3.1",
```

Comment:
did you add jquery, react helmet and react mailto on purpose, or are these left over from something else?

Reply:
@alodahl i believe this is from doing npm i ... was not intentional

Reply:
I just fixed a similar mistake of mine yesterday.  You can just:
1. go to the file history & find the version before this
2. choose "raw", copy the old version, and paste in into your branch package
3. delete your package-lock, npm i  again, then commit your new package and package lock.

Reply:
I can see that you made commits changing this file later, so I'm confused as to why it looks the same in this compare.

Reply:
Oh I see.  You still only changed web/package.json, but not package.json yet.